modules/gatsby/src/components/layout/top-level-layout.jsx

```diff
@@ -14,6 +14,7 @@ import SEO from '../common/SEO'
 import TableOfContents from './table-of-contents'
 import ExampleTableOfContents from './example-table-of-contents'
 import Header from './header'
+import { CompositeDisposable } from 'rx';
```

Comment:
nit: If choosing to import a component that is uncommon in our repos, maybe add a comment about what this is, to save the maintainer the burden of having to look it up.

For some reason I don't actually see it being used below...